cli/functionary/package.py

```diff
@@ -60,6 +61,13 @@ def create_cmd(ctx, language, name, output_directory):
 
     shutil.copytree(str(basepath), str(dir), dirs_exist_ok=True)
     generateYaml(output_directory, name, language)
+    click.echo(
+        "Next steps: \n"
```

Comment:
The formatting here is sufficiently bland that I completely missed the fact that this was displayed when I originally tested it out.  It just looks like a wall of text that my brain ignores by default.  Try to format it / punch it up in such a way that people will actually take notice of the text there and be less inclined to ignore it.

Also, the first line of the output (which was already there before your changes) is:

> Generating python function named mypackage

This doesn't feel like it tells us anything.  It's basically "Doing the thing you just asked me to do".  It probably makes sense to have the message come after the package is created and be a confirmation that the requested operation was successful.